Cython/Build/Cythonize.py

```diff
@@ -89,8 +89,12 @@ def _build(ext_modules, parallel):
         run_distutils(ext_modules[0])
         return
 
+    mp_configured_to_spawn = multiprocessing.get_start_method() == 'spawn'
+    if mp_configured_to_spawn:
+        print('Disabling parallel cythonization for "spawn" process start method.')
+
```

Comment:
I wonder if it's be better just to use an `mp_context` with a start method that does work rather than disabling it.

Reply:
@da-woods changed to `concurrent.futures` and checked that the `cythonize` cmd fails on mac and linux correctly.